### PR TITLE
Added kannada news headlines classification dataset. 

### PR DESCRIPTION
**Pull request by vrindaprabhu**
Manual Download of a kaggle dataset. Mostly followed process as ms_terms.

### CODE REVIEW

File: datasets/kannada_news/kannada_news.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    and manually download the dataset from Kaggle. Once it is completed,
+    a folder named archive.zip will appear in your Downloads folder(
+    or whichever folder your browser chooses to save files to). Unzip the folder to obtain
+    a folder named "archive" having train.csv and valid.csv.
+
+    You can then specify the path to this folder for the data_dir argument in the
+    datasets.load_dataset(...) option.
+
+    The <path/to/folder> can e.g. be "/Downloads/archive".
+    The data can then be loaded using the following command `datasets.load_dataset("kannada_news", data_dir="/Downloads/archive")`.
```
Comment (lhoestq) [inline datasets/kannada_news/kannada_news.py:57]: thanks for the clear instructions !

File: datasets/kannada_news/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+Though having roughly 38 million native speakers, it is a little under-represented language and will benifit from active contribution from the community.
+
+This dataset, however, can just help people get exposed to Kannada and help proceed further active participation for enabling continuous progress and development.
+
+
+## Additional Information
+
+### Dataset Curators
+
+[Gaurav Arora] (https://github.com/goru001/nlp-for-kannada) has also got some starter models an embeddings to help get started.
```
Comment (lhoestq) [inline datasets/kannada_news/README.md:104]: looks like some sections are missing

Could you re-add them please ? you can leave their content with `[More Information Needed]`

**Review by lhoestq — COMMENTED** [review]
Looking very good thanks !

I left a comment about the dataset card

File: datasets/kannada_news/README.md (hunk 2 above)
Comment (vrindaprabhu) [reply datasets/kannada_news/README.md]: Sorry! I have re-added them. Have left all the sections and filled up only the sections on which I could gather some info.

**Comment by vrindaprabhu** [comment]
Hi! Let me know if any more comments! Will fix it! :-)

File: datasets/kannada_news/README.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+There are starkingly less amount of data for South Indian languages, especially Kannada, available in digital format which can be used for NLP purposes.
+Though having roughly 38 million native speakers, it is a little under-represented language and will benifit from active contribution from the community.
```
Comment (lhoestq) [inline datasets/kannada_news/README.md:133]:
```suggestion
Though having roughly 38 million native speakers, it is a little under-represented language and will benefit from active contribution from the community.
```
typo

**Review by lhoestq — APPROVED** [review]
Thanks ! good job :)

**Pull request merged**